### PR TITLE
Remove extraneous packages from lock file

### PR DESCRIPTION
**Pull request by m-akinc**
# Pull Request

## 🤨 Rationale

Found old, extraneous entries in our lock file from before the Angular workspace got moved.

## 👩‍💻 Implementation

Removed them.

## 🧪 Testing

N/A

## ✅ Checklist

- [x] I have updated the project documentation to reflect my changes or determined no changes are needed.

### CODE REVIEW

**Comment by rajsite** [comment]
Can you comment on how they were removed? We shouldn't be hand editing the lock file

File: package-lock.json
Comment (m-akinc) [inline package-lock.json:1]: @rajsite 
> Can you comment on how they were removed? We shouldn't be hand editing the lock file

I hand edited it. ¯\\\_(ツ)\_/¯ 

In response to this comment, I went and tried to get `npm` to remove them for me (using `npm prune`), but apparently you can't. I found a couple open issues that seem to support this:
https://github.com/npm/cli/issues/5463
https://github.com/npm/cli/issues/3141

I tried removing the file and regenerating it from scratch, but of course that created a ton of changes, most of which were like this:
image
I thought maybe removing all `node_modules` directories and then running `npm i` again would restore the `"resolved"` and `"integrity"` values, but nope.

Comment (rajsite) [reply package-lock.json:1]: > I hand edited it. ¯\_(ツ)_/¯

Yea please don't hand edit the package-lock file.

We can rebuild the lockfile which will update dependencies at the same time and clear out cruft. ie. actually clean everything:

```
git clean -fdx
del package-lock.json
npm install
```

Was it just an annoyance or is it actually causing an issue that needs to be resolved? If it's causing an issue lets do the nuke and clean.

Comment (rajsite) [reply package-lock.json:1]: Did it real quick in the following PR: https://github.com/ni/nimble/pull/2123

On a xidax desktop it takes about three minutes to do the above commands.

**Comment by rajsite** [comment]
Closing in favor of #2123

**Pull request closed without merging**